content/blog/botan-first-milestone.md

```diff
@@ -0,0 +1,246 @@
++++
+title = "Botan: The first milestone"
+description = "A step towards improving the state of Haskell cryptography"
+date = 2024-06-03T12:15:00
+updated = 2024-06-03T12:15:00
+draft = false # Leave this as true so that it is not published
+template = "blog/page.html"
+
+[extra]
+lead = "After 8 months of work, we've reached an important milestone. Now we're here to tell you about what is coming next."
++++
+
+# Botan: The First Milestone
+
+First off, huge thanks to [Mercury][mercury] for funding the [first proposal][first proposal] and helping us reach this milestone - this library would not be in this state without their support. If you are an engineer looking for a savvy place to work, I hear they are [hiring][mercury hiring]!
+
+With [botan-bindings][botan-bindings] and [botan-low][botan-low] having reached their initial `0.0.1` release and `botan` making it to [package candidate][botan] status, we've reached an important milestone - the first released version!
+
+After 8 months of work, celebrations were in order, and after a bit of a breather, we're back in action to tell you about what is coming next.
+
+But first...
+
+# A call for users
+
+Do you use one of the following libraries?
+
+- `crypton` / `cryptonite`
+- `libsodium`
+- `saltine`
+
+What are you using cryptography for? Would you be interested in trying something new? Perhaps an alternate backend for your cryptography needs?
+
+We're working on `botan`, a cryptographic kitchen sink, and could use your feedback! We've successfully published `botan-bindings` and `botan-low`, and now are hard at work on getting `botan` ready too. That's where you come in! We can't listen to your feedback if there isn't any.
```

Comment:
This sentence could actually be the opening sentence. The blog post opens with a shout-out to Mercury and then talks about milestones, but you don't explain what it is until after.